Documentation/openshiftdocs/modules/metricsserverconfig.adoc

```diff
@@ -9,7 +9,7 @@
 
 === Description
 
-The `MetricsServerConfig` resource defines settings for the MetricsServer component.
+The `MetricsServerConfig` resource defines settings for the MetricsServer component. Note that this setting only applies when the MetricsServer feature gate is enabled.
```

Comment:
```suggestion
The `MetricsServerConfig` resource defines settings for the MetricsServer component. Note that this setting only applies when the MetricsServer feature gate is enabled.
```

For the component name, should it be "MetricsServer" or "Metrics Server" or "metrics server" or "metrics-server"? And is the feature gate name the same as the component name?

Reply:
The upstream project calls itself "Metrics Server" (https://github.com/kubernetes-sigs/metrics-server#kubernetes-metrics-server). The feature gate is `MetricsServer`.